### PR TITLE
Updated likes.md after the reactions bugs got patched

### PR DESCRIPTION
**Pull request by AlureonTheVirus**

### CODE REVIEW

File: members.md [hunk 1]
```diff
@@ -24,6 +24,46 @@ If the request succeeds, `meta.errors` will be null, and if the request fails, `
...
+        "id": "24681012",
+	"user_id": "11223344",
+	"name": "Alureon",
+	"nickname": "Alu",
+	"image_url": "https://i.groupme.com/1a3c5e7g",
+	"state": "active",
+	"roles": ["owner","admin"]
+      },
+      {
+      	"id": "1357911131",
```
Comment (not-so-smart) [inline members.md]: Could you make the indentation a bit more consistent?

File: likes.md [hunk 2]
```diff
@@ -42,15 +42,13 @@ Status: 200 OK
 
 ***
 
-## Reactions (Currently only on the v7 beta)
+## Reactions (Replacing likes in the new v7 update)
 
 React to a message with either a unicode or GroupMe emoji.
 
-Sending this POST request with no body results in a plain "❤️" reaction (It does NOT use the group's like icon as before in v6).
+As of writing this, web is the only client that renders any and all unicode emoji. The mobile apps are limited to rendering [GroupMe powerups](emoji.md) and the 15 options dysplayed in the client (❤️ 👍 🤣 🎉 🔥 😮 👀 😭 🥺 🙏 💀 🫶 🤬 💅 🫠). Any other unicode emojis will be accepted by the API but render as empty reactions in the conversation. It has been said that the full fluent emoji set by Microsoft may come in the future, however no hard deadlines or extra confirmation exists.
```
Comment (not-so-smart) [inline likes.md]: typo

```suggestion
As of writing this, web is the only client that renders any and all unicode emoji. The mobile apps are limited to rendering [GroupMe powerups](emoji.md) and the 15 options displayed in the client (❤️ 👍 🤣 🎉 🔥 😮 👀 😭 🥺 🙏 💀 🫶 🤬 💅 🫠). Any other unicode emojis will be accepted by the API but render as empty reactions in the conversation. It has been said that the full fluent emoji set by Microsoft may come in the future, however no hard deadlines or extra confirmation exists.
```

File: members.md [hunk 3]
```diff
@@ -24,6 +24,46 @@ If the request succeeds, `meta.errors` will be null, and if the request fails, `
 
 ***
 
+## Index Members
+
+Fetch a group's current or former member list.
```
Comment (not-so-smart) [inline members.md:29]: Isn't this endpoint only accessible by admins? If so, please include that as a note

File: members.md [hunk 4]
```diff
@@ -171,6 +211,32 @@ Status: 200 OK
 
 ***
 
+## Ban Member
```
Comment (not-so-smart) [inline members.md]: We need to make sure it's very clear that this endpoint is v2, it's very easy for someone to miss the note 


```suggestion
## Ban Member (v2)
```

**Review by not-so-smart — CHANGES_REQUESTED** [review]

File: members.md (hunk 1 above)
Comment (AlureonTheVirus) [reply members.md]: It should be 2 spaces every indent. I just struggle. I'll fix tonight.

Edit: I just figured out how to commit changes on mobile. These issues have been fixed.

File: members.md (hunk 3 above)
Comment (AlureonTheVirus) [reply members.md:29]: You're correct! I missed that I'm sorry! I'll update tonight.

File: members.md [hunk 5]
```diff
@@ -24,6 +24,48 @@ If the request succeeds, `meta.errors` will be null, and if the request fails, `
...
+{
+  "memberships":[
+    {
+      "id": "24681012",
+	     "user_id": "11223344",
+	     "name": "Alureon",
+	     "nickname": "Alu",
+	     "image_url": "https://i.groupme.com/1a3c5e7g",
+     	"state": "active",
+	     "roles": ["owner","admin"]
```
Comment (not-so-smart) [inline members.md]: formatting again

**Review by not-so-smart — CHANGES_REQUESTED** [review]

File: members.md (hunk 5 above)
Comment (AlureonTheVirus) [reply members.md:54]: I can't see this in my editor... It looks fine on my end. I'm not sure what's up.

**Pull request merged**